akka-docs/rst/scala/persistence-query.rst

```diff
@@ -0,0 +1,212 @@
+.. _persistence-query-scala:
+
+#################
+Persistence Query
+#################
+
```

Comment:
This page is not linked from anywhere. Probably should be added to `akka-docs/rst/scala/index-actors.rst`.